insights/parsers/lspci.py

```diff
@@ -69,14 +69,20 @@ def parse_content(self, content):
         # Parse kernel driver lines
         self.data = {}
         bus_device_function = ""
+
+        fields = ["Subsystem", "Kernel driver in use", "Kernel modules"]
+
         for line in get_active_lines(content):
             parts = line.split(None)
-            if parts[0] and parts[0] not in ['Subsystem:', 'Kernel'] \
+            if not parts:
+                continue
+
+            if parts[0] and parts[0] not in fields \
```

Comment:
@akshay196 I believe you can simplify this logic given that you have established that `parts` must have something in it in order to get here.  Also won't `parts[0]` just have the first word ('Subsystem:' and 'Kernel') due to the split on line 76? So `parts[0] not in fields` is not really useful here is it since it will always be `True`?

Reply:
@bfahr I have simplified this logic. Also `parts[0] not in fields` is removed which is not really necessary here. We are checking in fields in elif part later in the code. 

Thanks for pointing.